dev/NavigationView/NavigationView.xaml

```diff
@@ -927,7 +930,7 @@
                             x:Name="SeparatorLine"
                             Height="{ThemeResource NavigationViewItemSeparatorHeight}"
                             Fill="{ThemeResource NavigationViewItemSeparatorForeground}"
-                            Margin="{TemplateBinding Margin}"/>
+                            Margin="{ThemeResource NavigationViewItemSeparatorMargin}"/>
```

Comment:
what is the reason for this change?:)

Reply:
The template binding was causing the margins to be set twice. This sets the properly defined margins on the separator.